x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,44 @@
+<!--
```

Comment:
I guess ok for these to be Golang types or use protobuf for consistency?

Reply:
it seems cosmos has a pattern to use golang types in [state spec](https://github.com/cosmos/cosmos-sdk/blob/master/x/slashing/spec/02_state.md), but protobuf in [messages](https://github.com/cosmos/cosmos-sdk/blob/master/x/slashing/spec/03_messages.md).